src/project-impact-graph.yaml

```diff
@@ -2,11 +2,11 @@ globalExcludedGlobs:
     - OWNERS
     - build.sh
     - bootstrap.sh
-    - common/autoinstallers/**
+    - common/autoinstallers/
 projects:
     A:
         includedGlobs:
-            - projects/folder_A/**
+            - projects/folder_A/
```

Comment:
Could you elaborate the syntax used here?

Reply:
projects:
    A:
        includedGlobs:
            - projects/folder_A/
        excludedGlobs:
            - projects/folder_A/README.md
        dependentProjects:
            - G
This semantics describes a project named 'A', and 'includedGlobs' is used to specify the files that should be included in project 'A'. 'excludedGlobs' is a subset of 'includedGlobs', used to specify which paths in project 'A' need to be filtered. 'dependentProjects' indicates which projects directly depend on 'A'

'projects/folder_A/', Because we replaced glob match with `startsWith`, this folder path is used here to represent project A.

Reply:
It feels like it is no longer `included/excludedGlobs`. It's `included/excluedPrefix` now.

Reply:
You are right, it is no longer appropriate to continue using 'glob'